src/Sulu/Bundle/MediaBundle/DependencyInjection/Configuration.php

```diff
@@ -62,12 +62,26 @@ public function getConfigTreeBuilder()
                     ->arrayNode('default_imagine_options')
                         ->prototype('scalar')->end()
                     ->end()
+                    ->arrayNode('mime_types')
+                        ->prototype('scalar')->end()->defaultValue(array(
+                                                                'image/jpeg',
+                                                                'image/jpg',
+                                                                'image/gif',
+                                                                'image/png',
+                                                                'image/bmp',
+                                                                'image/svg+xml',
+                                                                'image/vnd.adobe.photoshop',
+                                                                'application/pdf',
+                                                            ))
+                    ->end()
                 ->end()
             ->end()
             ->arrayNode('format_cache')
                 ->addDefaultsIfNotSet()
                 ->children()
-                    ->scalarNode('public_folder')->defaultValue(SuluMediaExtension::DEFAULT_FORMAT_CACHE_PUBLIC_FOLDER)->end()
+                    ->scalarNode('path')->defaultValue('%assetic.write_to%/uploads/media')->end()
```

Comment:
@danrot @chirimoya 
so the best way I check if assetic parameter exists and else set it to default (web) and if it change he can overwrite it in the config.yml? Or do you know another way to get the symfony app public folder. In controllers where a request object exist `$this->get('request')->getBasePath();` will return the web/ folder which I need but on none requests I didnt found a way to get it.

I did also use the assetic bundle here: https://github.com/sulu-io/sulu/pull/951 maybe I should go here also on the old way to go over all bundles not only the assetic bundles?